server/models/query.js

```diff
@@ -0,0 +1,12 @@
+import sequelize from '../models';
+
+export const query = {
+  bulkCreateTags(tags) {
+    // NOTE: sequelize.model.bulkCreate fail on duplicate in postgres
+    return sequelize.sequelize.query(`INSERT INTO tag (name) VALUES
+            ${tags.map(tag => `('${tag}')`).join(', ')}
+            ON CONFLICT DO NOTHING;`);
```

Comment:
We need to use proper replacements provided by the sequelize.query: refs: http://docs.sequelizejs.com/manual/tutorial/raw-queries.html

Using JS template literals here have risks of SQL injection.